packages/vscode-extension/tsconfig.json

```diff
@@ -4,12 +4,6 @@
   "compilerOptions": {
     "outDir": "dist",
     "jsx": "react-jsx",
-    "types": [
-      "@wdio/mocha-framework",
-      "wdio-vscode-service",
-      "@wdio/globals/types",
-      "@wdio/types"
-    ],
-    "typeRoots": ["../../node_modules/@types", "../../node_modules"]
+    "types": ["@wdio/globals/types"]
```

Comment:
From the docs on wdio 8 with typescript

https://webdriver.io/docs/typescript/#framework-setup